stock_putaway_product/wizard/quick_add_location.py

```diff
@@ -0,0 +1,82 @@
+# coding: utf-8
+# © 2017 David BEAL @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, models, api, fields
+
+
+class QuickAddProductLocation(models.TransientModel):
+    _name = "quick.add.product.location"
+
+    def _default_putaway(self):
+        """ Overridable method
+        """
+        return self.env.ref(
+            "stock_putaway_product.product_putaway_per_product_wh",
+            raise_if_not_found=False,
+        )
+
+    def __default_putaway(self):
+        return self._default_putaway()
+
+    putaway_id = fields.Many2one(
+        comodel_name="product.putaway",
+        string="Put Away Method",
+        required=True,
+        default=__default_putaway,
```

Comment:
@PierrickBrun You can use lambda to avoid duplicate method call.

```suggestion
        default=lambda self: self._default_putaway(),
```